recipe/meta.yaml

```diff
@@ -34,13 +37,14 @@ requirements:
     - wheel >=0.37.0
     - importlib_resources
     - tbb-devel
+    - numpy ~=1.0
```

Comment:
Since this has not been using `numpy` in `host` so far, am guessing it is not needed here